util/configowner.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
This file should probably be moved under `bootstrap/util` instead, given that it seems very specific to what the Kubeadm bootstrapper needs.

wdyt?